systemtest/benchtest/expvar.go

```diff
@@ -74,3 +79,32 @@ func queryExpvar(out *expvar) error {
 	defer resp.Body.Close()
 	return json.NewDecoder(resp.Body).Decode(out)
 }
+
+// WaitUntilServerInactive blocks until one of the conditions occurs:
+// * APM Server output active events are lower than `warmupInactive`.
+// * HTTP call returns with an error
+// * Context is done.
+func WaitUntilServerInactive(ctx context.Context) error {
+	errChan := make(chan error)
+	go func() {
```

Comment:
What's the point of the goroutine if we're blocking on the result?

Reply:
I do not remember the intent, but I might've wanted to return the channel instead. Looking at it again it does make sense to not create this goroutine.